src/structs.py

```diff
@@ -157,6 +157,17 @@ def _copy_state_value(self, val: Any) -> Any:
         assert hasattr(val, "copy")
         return val.copy()
 
+    def allclose(self, other: State) -> bool:
```

Comment:
I had to grab this from master since my branch didn't have it yet...

Reply:
You'll have to rebase onto master before we can merge anyway, do you see the "Resolve conflicts" button at the bottom of this PR?

Reply:
I did that, now it's telling me to merge the latest from master into this branch. should I do that?

Reply:
yeah, you can always do that, this is your branch ;) it's just making sure your changes are applied on top of the most recent master

the latest from master would be bringing in my PR which you reviewed (#107)

Reply:
oh and so that means you won't need this allclose thing anymore